client/interfaces.go

```diff
@@ -313,6 +313,7 @@ type InstanceServer interface {
 	GetClusterMember(name string) (member *api.ClusterMember, ETag string, err error)
 	UpdateClusterMember(name string, member api.ClusterMemberPut, ETag string) (err error)
 	RenameClusterMember(name string, member api.ClusterMemberPost) (err error)
+	CreateClusterMember(args api.ClusterMembersPost) (op Operation, err error)
```

Comment:
nit but `args` should be replaced by `member` here as we use `args` for Args struct exclusively.